tech-cent/momo_bank_web
# #170528209 Create a landing page.

**What does this PR do?**
This PR adds a landing page to the momo bank web app

**Description of the task to be completed?**
- create a navigation bar
- create a footer
- create a body for the landing page

**How should this be manually tested?**
- Clone and set up the project using the instruction in the README.
- $ git checkout to `ft-create-landing-page-170528209` branch which contains the changes.
- Use $ npm start to start the application
The application should run with the landing page loaded in the browser

**What are the relevant pivotal tracker stories?**
[#170528209](https://www.pivotaltracker.com/story/show/170528209)

**Screenshots**
<img width="1249" alt="Screenshot 2020-01-08 at 20 39 54" src="https://user-images.githubusercontent.com/38077368/72001729-155a3b80-3257-11ea-884b-d507d75c5c50.png">
